pipenv/exceptions.py

```diff
@@ -62,13 +72,14 @@ def __init__(self, message=None, **kwargs):
     def show(self, file=None):
         if file is None:
             file = sys.stderr
+        console = Console(file=file)
         if self.extra:
             if isinstance(self.extra, str):
                 self.extra = [self.extra]
             for extra in self.extra:
                 extra = f"[pipenv.exceptions.{self.__class__.__name__}]: {extra}"
-                click.echo(extra, file=file)
-        click.echo(f"{self.message}", file=file)
+                console.print(extra)
+        console.print(f"{self.message}")
```

Comment:
i think this show function is unreachable, were you able to hit it?

Reply:
I have not seen an exception raised with extra in the code. But we might just want to leave it, as it should be used.